data/libs/SpaceStation.lua

```diff
@@ -60,17 +61,29 @@ local transientMarket = utils.automagic()
 
 local ensureStationData
 
+local function techLevelDiff(equip, station)
+	if equip == 'MILITARY' then equip = 11 end
+	if station == 'MILITARY' then station = 11 end
```

Comment:
This implies MILITARY tech level is equivalent to TL11 - is it possible for a non-military station to be TL11?

Might it be safer for MILITARY to be well outside the range, for example, 99, and ensure that non-military stations/equipment can never be that high?

Or possibly have "MILITARY" simply be an additional property, which might allow for multiple grades of military tech in the future?

Reply:
> is it possible for a non-military station to be TL11?

No, Military Tech Level is "11", yet it has equipment that is unique to it, i.e. TL12 does not have TL11/Military tech.